content/millennium/r4/general-clinical/condition.md

```diff
@@ -73,6 +73,63 @@ _Implementation Notes_
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Create
+
+Create a new Condition.
+
+    POST /Condition
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* Modifier fields should not be provided, and will cause the transaction to fail.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
```

Comment:
Do we need Accept header?

Reply:
removed: d6f7fbd16cf339fae076b5047a078bd4cdd50682